airbyte-integrations/connectors/source-notion/source_notion/schemas/databases.json

```diff
@@ -545,6 +545,28 @@
                     "type": "object"
                   }
                 }
+              },
+              {
```

Comment:
Please follow the reference doc for schema here: https://docs.airbyte.com/connector-development/schema-reference